extensions/wikia/ArticleVideo/scripts/ooyala-player.js

```diff
@@ -56,6 +58,10 @@ define('ooyala-player', [
 			self.onPlaybackReady();
 		});
 
+		messageBus.subscribe(OO.EVENTS.ADS_PLAYED, 'video-tracker', function () {
+			self.adIndex += 1;
```

Comment:
As an alternative, you can pass right context to the callback method using `.bind(this)`. Having that said `self` is already used in this file so it's ok, to leave it as it is ;)